graphql-server/src/schemas/queries/samples.ts

```diff
@@ -386,10 +389,15 @@ export async function queryDashboardSamples({
     const result = await session.run(samplesCypherQuery);
     return result.records.map((record) => {
       const recordObject = record.toObject().resultz;
+      const instrumentModel = getInstrumentModelByLibraries(
+        recordObject.libraries
+      );
       return {
         ...recordObject,
         cancerType: oncotreeCache?.[recordObject.oncotreeCode]?.mainType,
         cancerTypeDetailed: oncotreeCache?.[recordObject.oncotreeCode]?.name,
+        instrumentModel: instrumentModel,
+        platform: getPlatformByInstrumentModel(instrumentModel),
```

Comment:
I put this getter logic here instead of in `frontend/` for better separation of concerns. Ideally we'll keep the data processing logic in the backend resolver and leave the frontend for UI rendering if possible.